From a1d869384eb9ba1c7d0c82194ec7cf80574efc8a Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 06 Sep 2011 07:32:15 +0000
Subject: Fix typo in cogl/cogl-display.h

The line "#define cogl_display_get_rendrer cogl_display_get_rendrer_EXP"
should read
"#define cogl_display_get_renderer cogl_display_get_renderer_EXP"...

https://bugzilla.gnome.org/show_bug.cgi?id=658333

Reviewed-by: Emmanuele Bassi <ebassi@linux.intel.com>
---
diff --git a/cogl/cogl-display.h b/cogl/cogl-display.h
index 8cd215b..e0006a9 100644
--- a/cogl/cogl-display.h
+++ b/cogl/cogl-display.h
@@ -81,7 +81,7 @@ CoglDisplay *
 cogl_display_new (CoglRenderer *renderer,
                   CoglOnscreenTemplate *onscreen_template);
 
-#define cogl_display_get_rendrer cogl_display_get_rendrer_EXP
+#define cogl_display_get_renderer cogl_display_get_renderer_EXP
 CoglRenderer *
 cogl_display_get_renderer (CoglDisplay *display);
 
--
cgit v0.9.0.2
